### PR TITLE
305 Update navbar and create accessibility page

### PR DESCRIPTION
**Pull request by Israa91**
#### Issues this PR relates to #305 

=======

_NOTE: REMEMBER NOT TO REVIEW UNLESS ALL ACCEPTANCE CRITERIA HAS BEEN TICKED OFF_

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist

Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)

- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: client/src/pages/General/AccessibilityPage/index.js [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        </Row>
+        <Row mb="6" mbM="4">
+          <Col w={[4, 12, 12]}>
+            <T.P color="gray8" weight="bold">
+              Internet Explorer
+            </T.P>
+          </Col>
+          <Col w={[4, 12, 12]}>
+            <T.P color="gray8">
+              Go to “View” on the menu bar > Select text size / zoom
```
Comment (fadeomar) [inline client/src/pages/General/AccessibilityPage/index.js:48]:
```suggestion
              Go to “View” on the menu bar &gt; Select text size / zoom
```
invalid charcter

File: client/src/pages/General/AccessibilityPage/index.js [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+        </Row>
+        <Row mb="6" mbM="4">
+          <Col w={[4, 12, 12]}>
+            <T.P color="gray8" weight="bold">
+              Firefox
+            </T.P>
+          </Col>
+          <Col w={[4, 12, 12]}>
+            <T.P color="gray8">
+              Go to “View” on the menu bar > Select text size / zoom.
```
Comment (fadeomar) [inline client/src/pages/General/AccessibilityPage/index.js:60]:
```suggestion
              Go to “View” on the menu bar &gt; Select text size / zoom.
```

File: client/src/pages/General/AccessibilityPage/style.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+export const Link = styled(RouterLink)`
+  font-weight: 700;
+  text-decoration: underline !important;
+  color: ${theme.colors.blue} !important;
+`;
+export const A = styled.a`
+  font-weight: 700;
+  text-decoration: underline !important;
+  color: ${theme.colors.blue} !important;
+`;
```
Comment (fadeomar) [inline client/src/pages/General/AccessibilityPage/style.js:33]: this covers our Typograpghy common component as `Typograpghy.Link` and you can pass props like underline and color and external to achieve your goal here

File: client/src/pages/General/AccessibilityPage/index.js [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+          <Col w={[4, 12, 12]}>
+            <T.P color="gray8" weight="bold">
+              Apple
+            </T.P>
+          </Col>
+          <Col w={[4, 12, 12]}>
+            <T.P color="gray8">
+              <S.A href="https://www.google.fr/">
+                Click here to download Voice Aloud Reader
+              </S.A>{' '}
```
Comment (fadeomar) [inline client/src/pages/General/AccessibilityPage/index.js:147]:
```suggestion
              <T.Link external underline color="blue" to="https://www.google.fr/">
                Click here to download Voice Aloud Reader
              </T.Link>{' '}
```

File: client/src/pages/General/AccessibilityPage/index.js [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+        </Row>
+        <Row mb="6" mbM="4">
+          <Col w={[4, 12, 12]}>
+            <T.P color="gray8" weight="bold">
+              Android Apps
+            </T.P>
+          </Col>
+          <Col w={[4, 12, 12]}>
+            <T.P color="gray8">
+              <S.A href="https://www.google.fr/">
```
Comment (fadeomar) [inline client/src/pages/General/AccessibilityPage/index.js:130]:
```suggestion
              <T.Link external underline color="blue" to="https://www.google.fr/">
```

File: client/src/pages/General/AccessibilityPage/index.js [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+        </Row>
+        <Row mb="6" mbM="4">
+          <Col w={[4, 12, 12]}>
+            <T.P color="gray8" weight="bold">
+              Chrome
+            </T.P>
+          </Col>
+          <Col w={[4, 12, 12]}>
+            <T.P color="gray8">
+              <S.A href="https://www.google.fr/">
```
Comment (fadeomar) [inline client/src/pages/General/AccessibilityPage/index.js:102]:
```suggestion
              <T.Link external underline color="blue" to="https://www.google.fr/">
```

File: client/src/pages/General/AccessibilityPage/index.js [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+            Accessibility on this website is guided by government standards and
+            the Web Content Accessibility Guidelines WCAG are widely accepted as
+            the international standard for accessibility on the web. Whilst we
+            aim to make this website accessible to all users and achieve a
+            conformance level ‘AA’; we continually work with stakeholders to
+            ensure that conformance level ‘A’ is adhered to as a minimum. If you
+            experience any accessibility issue on this site or have any comment,
+            please
+            <S.Link to="/"> contact us</S.Link>.
+          </T.P>
```
Comment (fadeomar) [inline client/src/pages/General/AccessibilityPage/index.js:26]: this doesn't match the wireframe, those are three Paragraphs not one!!

File: client/src/pages/General/AccessibilityPage/index.js [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+          <Col w={[4, 12, 12]}>
+            <T.P color="gray8">
+              <S.A href="https://www.google.fr/">
+                Click here to download Read Aloud
+              </S.A>{' '}
+              for Android phones or tablets
+            </T.P>
+          </Col>
+        </Row>
+        <Row mb="6" mbM="4">
```
Comment (fadeomar) [inline client/src/pages/General/AccessibilityPage/index.js:137]: remove unused Row you can add the margin props to the Col component too

File: client/src/pages/General/AccessibilityPage/index.js [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+          <Col w={[4, 12, 12]}>
```
Comment (fadeomar) [inline client/src/pages/General/AccessibilityPage/index.js:138]:
```suggestion
          <Col w={[4, 12, 12]} mb="6" mbM="4">
```

File: client/src/pages/General/AccessibilityPage/index.js [hunk 10]
```diff
@@ -0,0 +1,158 @@
...
+          </Col>
+          <Col w={[4, 12, 12]}>
+            <T.P color="gray8">
+              Open the Edge browser and then click on Read Aloud Option or on
+              your keyboard press Ctrl + Shift + U
+            </T.P>
+          </Col>
+        </Row>
+        <Row mb="6" mbM="4">
+          <Col w={[4, 12, 12]}>
```
Comment (fadeomar) [inline client/src/pages/General/AccessibilityPage/index.js:123]:
```suggestion
          <Col w={[4, 12, 12]}  mb="6" mbM="4">
```
remove unused Row you can add the margin props to the Col component too

File: client/src/pages/General/AccessibilityPage/index.js [hunk 11]
```diff
@@ -0,0 +1,158 @@
...
+            <T.P color="gray8">
+              <S.A href="https://www.google.fr/">
+                Click here to download Google Speak
+              </S.A>{' '}
+              and select the Add to Chrome button
+            </T.P>
+          </Col>
+        </Row>
+        <Row mb="6" mbM="4">
+          <Col w={[4, 12, 12]}>
```
Comment (fadeomar) [inline client/src/pages/General/AccessibilityPage/index.js:110]:
```suggestion
          <Col w={[4, 12, 12]} mb="6" mbM="4">
```
remove unused Row you can add the margin props to the Col component too

File: client/src/pages/General/AccessibilityPage/index.js [hunk 12]
```diff
@@ -0,0 +1,158 @@
...
+        <Row mb="6" mbM="4">
+          <T.P color="gray8">
+            Many computers and mobile devices today have built in text-to-speech
+            software. Here are guides for each of the major browsers and
+            devices:
+          </T.P>
+        </Row>
+        <Row mb="6" mbM="4">
+          <Col w={[4, 12, 12]}>
+            <T.P color="gray8" weight="bold">
```
Comment (fadeomar) [inline client/src/pages/General/AccessibilityPage/index.js:96]:
```suggestion
        <Row >
          <Col w={[4, 12, 12]} mb="6" mbM="4">
            <T.P color="gray8" weight="bold">
```
remove unused Row you can add the margin props to the Col component too

File: client/src/pages/General/AccessibilityPage/index.js [hunk 13]
```diff
@@ -0,0 +1,158 @@
...
+              browser version.
+            </T.P>
+          </Col>
+        </Row>
+      </Row>
+      <Row mb="6" mbM="4">
+        <Line />
+      </Row>
+      <Row mb="6" mbM="4">
+        <Row mb="6" mbM="4">
```
Comment (fadeomar) [inline client/src/pages/General/AccessibilityPage/index.js:80]: too many Rows

File: client/src/pages/General/AccessibilityPage/index.js [hunk 14]
```diff
@@ -0,0 +1,158 @@
...
+            ensure that conformance level ‘A’ is adhered to as a minimum. If you
+            experience any accessibility issue on this site or have any comment,
+            please
+            <S.Link to="/"> contact us</S.Link>.
+          </T.P>
+        </Col>
+      </Row>
+      <Row mb="6" mbM="4">
+        <Line />
+      </Row>
```
Comment (fadeomar) [inline client/src/pages/General/AccessibilityPage/index.js:31]: the only reason to use `Row` component is to wrap Col`s component, you can render the Line component without putting it on Row and add the margin value to it directly

**Review by fadeomar — CHANGES_REQUESTED** [review]
hey @Israa91 nice work, but there is some points need to fix : 
  - [x] there is padding wrapping the all page not like the other pages, you can see the different on the wireframes .
image

  - [x] you must add target="_blank" for the external links to open them on a new tab and not lose the app 
  - [ ] when you add external links please add them to the constants file and import them as variables you can find on src > constants > nav-routes > EXTERNAL

File: client/src/pages/General/AccessibilityPage/index.js (hunk 1 above)
Comment (Israa91) [reply client/src/pages/General/AccessibilityPage/index.js:48]: Done

File: client/src/pages/General/AccessibilityPage/index.js (hunk 2 above)
Comment (Israa91) [reply client/src/pages/General/AccessibilityPage/index.js:60]: Done

File: client/src/pages/General/AccessibilityPage/index.js (hunk 5 above)
Comment (Israa91) [reply client/src/pages/General/AccessibilityPage/index.js:130]: Done

File: client/src/pages/General/AccessibilityPage/index.js (hunk 6 above)
Comment (Israa91) [reply client/src/pages/General/AccessibilityPage/index.js:102]: Done

File: client/src/pages/General/AccessibilityPage/index.js (hunk 14 above)
Comment (Israa91) [reply client/src/pages/General/AccessibilityPage/index.js:31]: Done

File: client/src/pages/General/AccessibilityPage/index.js (hunk 13 above)
Comment (Israa91) [reply client/src/pages/General/AccessibilityPage/index.js:80]: Done

File: client/src/pages/General/AccessibilityPage/index.js (hunk 12 above)
Comment (Israa91) [reply client/src/pages/General/AccessibilityPage/index.js:96]: Done

File: client/src/pages/General/AccessibilityPage/index.js (hunk 11 above)
Comment (Israa91) [reply client/src/pages/General/AccessibilityPage/index.js:110]: Done

File: client/src/pages/General/AccessibilityPage/index.js (hunk 10 above)
Comment (Israa91) [reply client/src/pages/General/AccessibilityPage/index.js:123]: Done

File: client/src/pages/General/AccessibilityPage/index.js (hunk 9 above)
Comment (Israa91) [reply client/src/pages/General/AccessibilityPage/index.js:138]: Done

File: client/src/pages/General/AccessibilityPage/index.js (hunk 8 above)
Comment (Israa91) [reply client/src/pages/General/AccessibilityPage/index.js:137]: Done

File: client/src/pages/General/AccessibilityPage/index.js (hunk 7 above)
Comment (Israa91) [reply client/src/pages/General/AccessibilityPage/index.js:26]: Done

File: client/src/components/Navbar/style.js [hunk 15]
```diff
@@ -50,3 +54,28 @@ export const RainbowContainer = styled.div`
   right: 0;
   z-index: 1;
 `;
+export const Collapse = styled(AntCollapse)`
+  margin-left: -38px;
```
[the remark this answers is not in the document]
Comment (Israa91) [reply client/src/components/Navbar/style.js]: Done

File: client/src/components/Navbar/style.js [hunk 16]
```diff
@@ -50,3 +54,28 @@ export const RainbowContainer = styled.div`
   right: 0;
   z-index: 1;
 `;
+export const Collapse = styled(AntCollapse)`
+  margin-left: -38px;
+`;
+export const AntPanel = styled(Panel)`
+  background-color: ${theme.colors.gray9};
+  color: ${theme.colors.white};
+  width: 377px;
```
[the remark this answers is not in the document]
Comment (Israa91) [reply client/src/components/Navbar/style.js]: Done

File: client/src/components/Navbar/style.js [hunk 17]
```diff
@@ -50,3 +54,28 @@ export const RainbowContainer = styled.div`
...
+  background-color: ${theme.colors.gray9};
+  color: ${theme.colors.white};
+  width: 377px;
+  height: 52px;
+  padding-left: 35px;
+
+  .ant-collapse-content-box {
+    background-color: ${theme.colors.gray9};
+    color: ${theme.colors.white};
+    margin-left: -35px;
```
[the remark this answers is not in the document]
Comment (Israa91) [reply client/src/components/Navbar/style.js]: Done

File: client/src/pages/General/AccessibilityPage/style.js (hunk 3 above)
Comment (Israa91) [reply client/src/pages/General/AccessibilityPage/style.js:33]: Done

File: client/src/components/AccessibilityBar/index.js [hunk 18]
```diff
@@ -0,0 +1,22 @@
+import * as S from './style';
+import Icon from '../Icon';
+import Button from '../Button';
+import { navRoutes } from '../../constants';
+const AccessibilityBar = () => {
+  return (
+    <S.Div1>
```
Comment (thejoefriel) [inline client/src/components/AccessibilityBar/index.js:7]: Not something to change here but in future think about more descriptive names for styled elements (e.g. Container, InnerContent)

File: client/src/components/Navbar/style.js [hunk 19]
```diff
@@ -50,3 +53,38 @@ export const RainbowContainer = styled.div`
   right: 0;
   z-index: 1;
 `;
+export const Collapse = styled(AntCollapse)`
+  padding: inherit;
```
Comment (thejoefriel) [inline client/src/components/Navbar/style.js:57]: do display flex and align-items center and this should vertically align the contents.  You'll notice that the arrow icon isn't quite in the center vertically as it should be

**Review by thejoefriel — CHANGES_REQUESTED** [review]
- [ ] navigation menu is missing from the accessibility page. 
image

**Comment by fadeomar** [comment]
some notes no fix: 
  - [ ] spacing on mobile 
image
  - [ ] clicking contact us direct me to `http://localhost:3000/digital@chilternmusictherapy.co.uk`

**Review by fadeomar — APPROVED** [review]

**Pull request merged**